### DISCUSSION TITLE
Client credentials flow support for "sf org login jwt"

### DISCUSSION THREAD

**Discussion by adambestsf** [forcedotcom/cli#2342] [Ideas]
**Is your feature request related to a problem? Please describe.**
To simplify CICD successfully logging into 100s of sandboxes or production, 1 less parameter to pass into the sf org login jwt command is always a win

**What are you trying to do**
Login to a SF Sandbox or production using "sf org login jwt" with Client Credentials flow. This does not require the a username. https://developer.salesforce.com/blogs/2023/03/using-the-client-credentials-flow-for-easier-api-authentication

Client credentials flow assumes running user set on connected app so you shouldn't have to pass a username into the sf command.

**Describe the solution you'd like**
I would like the flag "username" not mandatory on the sf org login jwt. This "should" allow a successful login with a connected app which has client credentials flow configured correctly.  1) client credentials flow ticked 2) Client Credentials Flow user completed

**Describe alternatives you've considered**
Continue to pass the flag "username" when connecting to a sandbox or production.

**Additional context**
Sample Error being received: Missing required flag username

sf org login jwt --jwt-key-file ...server.key --client-id xxxx --alias serviceqa --set-default --instance-url https://xxx.sandbox.my.salesforce.com/
Error (1): The following error occurred:
  Missing required flag username
See more help with --help

[a comment by an automated account is not shown]

**Comment by marsson** [comment]
AWESOME suggestion...maybe this would fall out of the JWT command, but a specific client logibn command would be of great value!!
